tpgtools/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/GoogleCloudPlatform/magic-modules/tpgtools
+
+go 1.13
+
+require (
```

Comment:
Oh weird! Does the DCL not appear here?

Reply:
Nope, we never use it in the generator, only in the templates it outputs